mne/event.py

```diff
@@ -209,7 +210,12 @@ def read_events(filename, include=None, exclude=None, mask=0):
         the exclude parameter is ignored.
     mask : int | None
         The value of the digital mask to apply to the stim channel values.
-        The default value is 0. ``None`` skips masking.
+        The default value is None. If None, trigger masking is skipped.
+    mask_type: 'and' | 'not_and'
+        The type of operation between the mask and the trigger.
+        Choose 'and' for MNE-C masking behavior.
```

Comment:
Most of the time people will probably learn about `mask` in `find_events` anyway, I think. I'll improve the links to that doc so we don't duplicate